### DISCUSSION TITLE
Unable to login as well as sign up to vercel

### DISCUSSION THREAD

**Discussion by baishakhi-nandi** [vercel/community#4161] [Account Issue] [answered]
### Question

Why am I unable to login or signup to the Vercel platform?

**Comment by amyegan** [comment, marked as answer]
Sorry you're having trouble signing up for a new account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
